### DISCUSSION TITLE
Feature request: Controller passthrough

### DISCUSSION THREAD

**Discussion by Ryhon0** [Genymobile/feedback#22] [Genymotion Desktop] [answered]
Currently if a controller is connected, the VM does not get any input from that controller. If this feature was added, Genymotion would connect a generic USB HID controller to the VM and pass the controller input to the VM. This would be especially useful on the SteamDeck because there are no android emulators with gamepad support for SteamOS

**Comment by jmaneyrol69** [comment, marked as answer]
@Ryhon0 a new Gamepad feature, which allows to connect a controller via USB or Bluetooth, will be implemented in the next stable release of Genymotion Desktop (v.3.8.0).
